app/presenters/finder_presenter.rb

```diff
@@ -11,15 +11,18 @@ def initialize(content_item, values = {}, keywords = nil)
     @organisations = content_item.links.organisations
     facets.values = values
     @keywords = keywords
-    @beta_message = content_item.details.beta_message
   end
 
   def beta?
-    content_item.details.beta
+    content_item.details.beta || false
```

Comment:
Nothing, but I thought it would be better to have `beta?` to explicitly return false than `nil`.